client/components/learn/Resources/resources-model.ts

```diff
@@ -323,5 +353,10 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://github.dev',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/github-dev_500w.png`,
     },
+    {
+      title: 'RegExr',
```

Comment:
Please replace with https://regex101.com/ - I've been using this one for a long time and I can attest to it being a good tool.

Also, let's move under `EXTRA` -  I don't think performance is a good category.